### PR TITLE
Scheduler

### PR DESCRIPTION
**Pull request by julmon**
Replace internal scheduler implementation by `temboard-sched`

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -13,6 +13,7 @@
     license='PostgreSQL',
     url='http://temboard.io/',
     description='Administration & monitoring PostgreSQL agent.',
+    install_requires=['temboard-sched>=1.0.dev1'],
```
Comment (bersace) [inline setup.py]: Pas du submodule finalement ?

File: temboardagent/api.py [hunk 2]
```diff
@@ -221,8 +217,7 @@ def logout(http_context, queue_in=None, config=None, sessions=None,
 
 
 @add_route('GET', '/discover')
-def get_discover(http_contexte, queue_in=None, config=None, sessions=None,
-                 commands=None):
+def get_discover(http_contexte, config=None, sessions=None):
```
Comment (pgiraud) [inline temboardagent/api.py]: (Not related to this PR) `http_context` ?

**Comment by julmon** [comment]
At this point functional tests look fine:
``` console
=================================== test session starts ====================================
platform linux2 -- Python 2.7.5, pytest-3.3.2, py-1.5.2, pluggy-0.6.0 -- /usr/bin/python2
cachedir: ../../.cache
rootdir: /home/julien/temboard-agent_julmon, inifile:
collected 93 items                                                                         

test_activity.py::TestActivity::test_00_env_pg PASSED                                [  1%]
test_activity.py::TestActivity::test_01_activity_root PASSED                         [  2%]
test_activity.py::TestActivity::test_02_activity_kill PASSED                         [  3%]
test_activity.py::TestActivity::test_03_activity_waiting PASSED                      [  4%]
test_activity.py::TestActivity::test_04_activity_blocking PASSED                     [  5%]
test_api.py::TestAPI::test_00_env_pg PASSED                                          [  6%]
test_api.py::TestAPI::test_01_login_ok PASSED                                        [  7%]
test_api.py::TestAPI::test_02_login_ok PASSED                                        [  8%]
test_api.py::TestAPI::test_03_login_ko PASSED                                        [  9%]
test_api.py::TestAPI::test_04_login_ko PASSED                                        [ 10%]
test_api.py::TestAPI::test_05_login_ko PASSED                                        [ 11%]
test_api.py::TestAPI::test_06_login_ko PASSED                                        [ 12%]
test_dashboard.py::TestDashboard::test_00_env_pg PASSED                              [ 13%]
test_dashboard.py::TestDashboard::test_01_dashboard_ok PASSED                        [ 15%]
test_dashboard.py::TestDashboard::test_02_dashboard_buffers_ok PASSED                [ 16%]
test_dashboard.py::TestDashboard::test_03_dashboard_hitratio_ok PASSED               [ 17%]
test_dashboard.py::TestDashboard::test_04_dashboard_active_backends_ok PASSED        [ 18%]
test_dashboard.py::TestDashboard::test_05_dashboard_cpu_ok PASSED                    [ 19%]
test_dashboard.py::TestDashboard::test_06_dashboard_loadaverage_ok PASSED            [ 20%]
test_dashboard.py::TestDashboard::test_07_dashboard_memory_ok PASSED                 [ 21%]
test_dashboard.py::TestDashboard::test_08_dashboard_hostname_ok PASSED               [ 22%]
test_dashboard.py::TestDashboard::test_09_dashboard_os_version_ok PASSED             [ 23%]
test_dashboard.py::TestDashboard::test_10_dashboard_databases_ok PASSED              [ 24%]
test_dashboard.py::TestDashboard::test_11_dashboard_pg_version_ok PASSED             [ 25%]
test_dashboard.py::TestDashboard::test_12_dashboard_n_cpu_ok PASSED                  [ 26%]
test_monitoring.py::TestMonitoring::test_00_env_pg PASSED                            [ 27%]
test_monitoring.py::TestMonitoring::test_01_monitoring_session PASSED                [ 29%]
test_monitoring.py::TestMonitoring::test_02_monitoring_xacts PASSED                  [ 30%]
test_monitoring.py::TestMonitoring::test_03_monitoring_locks PASSED                  [ 31%]
test_monitoring.py::TestMonitoring::test_04_monitoring_blocks PASSED                 [ 32%]
test_monitoring.py::TestMonitoring::test_05_monitoring_bgwriter PASSED               [ 33%]
test_monitoring.py::TestMonitoring::test_06_monitoring_db_size PASSED                [ 34%]
test_monitoring.py::TestMonitoring::test_07_monitoring_tblspc_size PASSED            [ 35%]
test_monitoring.py::TestMonitoring::test_08_monitoring_filesystems_size PASSED       [ 36%]
test_monitoring.py::TestMonitoring::test_09_monitoring_cpu PASSED                    [ 37%]
test_monitoring.py::TestMonitoring::test_10_monitoring_process PASSED                [ 38%]
test_monitoring.py::TestMonitoring::test_11_monitoring_memory PASSED                 [ 39%]
test_monitoring.py::TestMonitoring::test_12_monitoring_loadavg PASSED                [ 40%]
test_monitoring.py::TestMonitoring::test_13_monitoring_wal_files PASSED              [ 41%]
test_monitoring.py::TestMonitoring::test_14_monitoring_replication PASSED            [ 43%]
test_pgconf.py::TestPgconf::test_00_env_pg PASSED                                    [ 44%]
test_pgconf.py::TestPgconf::test_01_get_pgconf_configuration_ok PASSED               [ 45%]
test_pgconf.py::TestPgconf::test_02_get_pgconf_configuration_ko_401 PASSED           [ 46%]
test_pgconf.py::TestPgconf::test_03_get_pgconf_configuration_ko_406 PASSED           [ 47%]
test_pgconf.py::TestPgconf::test_04_post_pgconf_configuration_ok PASSED              [ 48%]
test_pgconf.py::TestPgconf::test_05_post_pgconf_configuration_ko_401 PASSED          [ 49%]
test_pgconf.py::TestPgconf::test_06_get_pgconf_configuration_ko_406 PASSED           [ 50%]
test_pgconf.py::TestPgconf::test_07_post_pgconf_configuration_ko_400 PASSED          [ 51%]
test_pgconf.py::TestPgconf::test_08_get_pgconf_configuration_status_ok PASSED        [ 52%]
test_pgconf.py::TestPgconf::test_09_get_pgconf_configuration_status_ko_406 PASSED    [ 53%]
test_pgconf.py::TestPgconf::test_10_get_pgconf_configuration_status_ko_401 PASSED    [ 54%]
test_pgconf.py::TestPgconf::test_11_get_pgconf_configuration_categories PASSED       [ 55%]
test_pgconf.py::TestPgconf::test_12_get_pgconf_configuration_categories_ko_401 PASSED [ 56%]
test_pgconf.py::TestPgconf::test_13_get_pgconf_configuration_categories_ko_406 PASSED [ 58%]
test_pgconf.py::TestPgconf::test_14_get_pgconf_configuration_category_ok PASSED      [ 59%]
test_pgconf.py::TestPgconf::test_15_get_pgconf_configuration_category_ko_401 PASSED  [ 60%]
test_pgconf.py::TestPgconf::test_16_get_pgconf_configuration_category_ko_406 PASSED  [ 61%]
test_pgconf.py::TestPgconf::test_17_get_pgconf_hba_ok PASSED                         [ 62%]
test_pgconf.py::TestPgconf::test_18_get_pgconf_hba_ko_401 PASSED                     [ 63%]
test_pgconf.py::TestPgconf::test_19_get_pgconf_hba_ko_406 PASSED                     [ 64%]
test_pgconf.py::TestPgconf::test_20_get_pgconf_hba_raw_ok PASSED                     [ 65%]
test_pgconf.py::TestPgconf::test_21_get_pgconf_hba_raw_ko_401 PASSED                 [ 66%]
test_pgconf.py::TestPgconf::test_22_get_pgconf_hba_raw_ko_406 PASSED                 [ 67%]
test_pgconf.py::TestPgconf::test_23_post_pgconf_hba_ok PASSED                        [ 68%]
test_pgconf.py::TestPgconf::test_24_post_pgconf_hba_ko_401 PASSED                    [ 69%]
test_pgconf.py::TestPgconf::test_25_post_pgconf_hba_ko_406 PASSED                    [ 70%]
test_pgconf.py::TestPgconf::test_26_post_pgconf_hba_ko_406 PASSED                    [ 72%]
test_pgconf.py::TestPgconf::test_27_post_pgconf_hba_version_ok PASSED                [ 73%]
test_pgconf.py::TestPgconf::test_28_get_pgconf_hba_versions_ok PASSED                [ 74%]
test_pgconf.py::TestPgconf::test_29_get_pgconf_hba_versions_ko_401 PASSED            [ 75%]
test_pgconf.py::TestPgconf::test_30_get_pgconf_hba_versions_ko_406 PASSED            [ 76%]
test_pgconf.py::TestPgconf::test_31_get_pgconf_hba_version_ok PASSED                 [ 77%]
test_pgconf.py::TestPgconf::test_32_get_pgconf_hba_version_ko_404 PASSED             [ 78%]
test_pgconf.py::TestPgconf::test_33_get_pgconf_hba_version_ko_406 PASSED             [ 79%]
test_pgconf.py::TestPgconf::test_34_get_pgconf_hba_raw_version_ok PASSED             [ 80%]
test_pgconf.py::TestPgconf::test_35_get_pgconf_hba_raw_version_ko_404 PASSED         [ 81%]
test_pgconf.py::TestPgconf::test_36_get_pgconf_hba_raw_version_ko_406 PASSED         [ 82%]
test_pgconf.py::TestPgconf::test_37_delete_pgconf_hba_version_ok PASSED              [ 83%]
test_pgconf.py::TestPgconf::test_38_delete_pgconf_hba_version_ko_401 PASSED          [ 84%]
test_pgconf.py::TestPgconf::test_39_delete_pgconf_hba_version_ko_406 PASSED          [ 86%]
test_pgconf.py::TestPgconf::test_40_delete_pgconf_hba_version_ko_406 PASSED          [ 87%]
test_pgconf.py::TestPgconf::test_41_delete_pgconf_hba_version_ko_404 PASSED          [ 88%]
test_pgconf.py::TestPgconf::test_42_get_pgconf_pg_ident_ok PASSED                    [ 89%]
test_pgconf.py::TestPgconf::test_43_get_pgconf_pg_ident_ko_401 PASSED                [ 90%]
test_pgconf.py::TestPgconf::test_44_get_pgconf_pg_ident_ko_406 PASSED                [ 91%]
test_pgconf.py::TestPgconf::test_45_post_pgconf_hba_raw_ok PASSED                    [ 92%]
test_pgconf.py::TestPgconf::test_46_post_pgconf_hba_raw_ko_401 PASSED                [ 93%]
test_pgconf.py::TestPgconf::test_47_post_pgconf_hba_raw_ko_406 PASSED                [ 94%]
test_pgconf.py::TestPgconf::test_48_post_pgconf_hba_raw_ko_406 PASSED                [ 95%]
test_pgconf.py::TestPgconf::test_49_post_pgconf_pg_ident_ok PASSED                   [ 96%]
test_pgconf.py::TestPgconf::test_50_post_pgconf_pg_ident_ko_401 PASSED               [ 97%]
test_pgconf.py::TestPgconf::test_51_post_pgconf_pg_ident_ko_406 PASSED               [ 98%]
test_pgconf.py::TestPgconf::test_52_post_pgconf_pg_ident_ko_406 PASSED               [100%]

================================ 93 passed in 35.90 seconds ================================
```

**Comment by julmon** [comment]
Green lights are back.

**Pull request merged**